.expeditor/config.yml

```diff
@@ -106,6 +70,33 @@ staging_areas:
       workload: pull_request_merged:chef/automate:master:*
 
 subscriptions:
+  # dev -> acceptance promotion
+  # These actions are taken when `/expeditor promote` is run from Slack
+  - workload: project_promoted:{{agent_id}}:dev:*
+    actions:
+    - bash:.expeditor/promote-hab-pkgs-and-cli.sh:
+        post_commit: true
+    - purge_packages_chef_io_fastly:{{target_channel}}/automate/latest:
+        post_commit: true
+    - bash:.expeditor/announce-acceptance.sh:
+        post_commit: true
+    - trigger_pipeline:deploy/acceptance
+    - trigger_pipeline:deploy/automate-chef-io
```

Comment:
I believe this was running on all promotions before.  Is this really what we want? Do we need to deploy automate.chef.io on dev->acceptance promotions?